### ISSUE TITLE
All embedded license files are in a single line

### ISSUE THREAD

**Issue by mpe85** [jk1/Gradle-License-Report#146]
If there are multiple embedded license files found in a library, they are printed in a single line, not even separated by a whitespace which looks a bit odd.

E.g.
`jetty-client-9.4.19.v20190610.jar/META-INF/LICENSEjetty-client-9.4.19.v20190610.jar/META-INF/LICENSEjetty-client-9.4.19.v20190610.jar/META-INF/NOTICE.txt`

Each link should be on a separate line, or at least they should be separated with a comma or semicolon:

```
jetty-client-9.4.19.v20190610.jar/META-INF/LICENSE
jetty-client-9.4.19.v20190610.jar/META-INF/LICENSE
jetty-client-9.4.19.v20190610.jar/META-INF/NOTICE.txt
```
or
`jetty-client-9.4.19.v20190610.jar/META-INF/LICENSE; jetty-client-9.4.19.v20190610.jar/META-INF/LICENSE; jetty-client-9.4.19.v20190610.jar/META-INF/NOTICE.txt`

**Comment by jk1** [comment]
@mpe85 please share your `licenseReport {...}` configuration block

**Comment by mpe85** [comment]
```
licenseReport {
  renderers = arrayOf(InventoryHtmlReportRenderer("report.html", "Backend"))
  filters = arrayOf<DependencyFilter>(LicenseBundleNormalizer("$projectDir/license-normalizer-bundle.json", true))
}
```

**Comment by guenhter** [comment]
To narrow this down quickly: Which concrete dependency causes this problem?

**Comment by guenhter** [comment]
Ah, forget it. Silly me. `jetty-client-9.4.` alright

**Comment by mpe85** [comment]
Yes, but it also happens with these:
- org.apache.velocity:velocity-engine-core:2.1
- com.fasterxml.jackson.core:jackson-*:2.9.9
- com.fasterxml.jackson.module:jackson-*:2.9.9
- org.xerial:sqlite-jdbc:3.28.0

**Comment by jk1** [comment]
Merged and released in the scope of 1.9
